model.py

```diff
@@ -10039,6 +10039,7 @@ class ExternalIntegration(Base, HasFullTableCache):
     ONE_CLICK = RB_DIGITAL
     OPDS_FOR_DISTRIBUTORS = u'OPDS for Distributors'
     ENKI = DataSource.ENKI
+    FEEDBOOKS = DataSource.FEEDBOOKS
```

Comment:
Although Feedbooks titles are imported via OPDS, the import procedure is significantly different from a normal OPDS import, to the extent that we treat it as a different protocol with its own configuration settings.